index.js

```diff
@@ -32,7 +32,8 @@ const defaultOptions = {
   waitFor: false,
   externalServer: false,
   // workaround for https://github.com/geelen/react-snapshot/issues/66#issuecomment-331718560
-  fixWebpackChunksIssue: false, // experimental
+  fixWebpackChunksIssue: true, // experimental
+  bundleName: "main",
```

Comment:
there are some changes in this code in https://github.com/stereobooster/react-snap/pull/41. There will be conflicts

Reply:
so which is the latest branch that I should pull?

Reply:
I suppose you can do PR against `breaking-changes` branch. It is pretty huge right now I do not want to introduce merge conflicts

Reply:
So should I merge the breaking-changes branch into my branch?

Reply:
can you reintroduce changes to new branch based on `breaking-changes`. Merge will be messy.

Reply:
Why did you remove the 404 route from include?

Reply:
it is added automatically if there is more than one page